gamerule.lua

```diff
@@ -0,0 +1,188 @@
+
+-- gamerule.lua
+
+-- Implements gamerule command and related functions
+
+
+
+-- SQLite DB handler
+local grDB
+local ini = cIniFile()
+local boolean = "boolean"
+local number = "number"
```

Comment:
These two lines are rather weird, can you change the name of the variables to something more descriptive of what they're used for?